virtManager/delete.py

```diff
@@ -493,3 +491,25 @@ def append_str(str1, str2, delim="\n"):
         log.exception("Failed checking disk conflict: %s", str(e))
 
     return (not info, info)
+
+
+class vmmDeleteDialog(_vmmDeleteBase):
+
```

Comment:
Please move these classes up to be right after the _vmmDeleteBase class, before the 'UI init helpers' comment block
Also remove the extra newline here, before `__init__(`